recipes/libtool/all/conanfile.py

```diff
@@ -57,17 +59,25 @@ def _configure_autotools(self):
         conf_args = [
             "--datarootdir={}".format(datarootdir),
             "--prefix={}".format(prefix),
-            "--disable-shared",
-            "--disable-static",
+            "--enable-shared",
+            "--enable-static",
             "--disable-ltdl-install",
         ]
         self._autotools.configure(args=conf_args, configure_dir=self._source_subfolder)
         return self._autotools
 
+    def _patch_sources(self):
+        # FIXME: tools.patch cannot patch read-only files on Windows
+        os.chmod(os.path.join(self._source_subfolder, "build-aux", "ltmain.sh"), 0o777)
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        os.chmod(os.path.join(self._source_subfolder, "build-aux", "ltmain.sh"), 0o555)
+
     def build(self):
+        self._patch_sources()
         with self._build_context():
             autotools = self._configure_autotools()
-            autotools.make()
+            autotools.make(args=["-j1"])
```

Comment:
it's okay, but for the future, I think make has **parallel** option

Reply:
Oh, this is actually not needed but a leftover from debugging.